src/main/java/com/yahoo/sketches/concurrent/theta/UpdateSketchComposition.java

```diff
@@ -0,0 +1,107 @@
+package com.yahoo.sketches.concurrent.theta;
+
+import com.yahoo.memory.WritableMemory;
+import com.yahoo.sketches.Family;
+import com.yahoo.sketches.ResizeFactor;
+import com.yahoo.sketches.theta.UpdateReturnState;
+import com.yahoo.sketches.theta.UpdateSketch;
+
```

Comment:
External comment please - why do we need this class

Reply:
I will add documentation.
Basically this class aims to eliminate code duplication. Currently 2 classes extend it and don't need to replicate the code in it.